Reflektion.tpl

```diff
@@ -0,0 +1,503 @@
+﻿___INFO___
+
+{
+  "displayName": "Reflektion",
+  "description": "Template for setting up beacon, context and events",
+  "securityGroups": [],
+  "id": "cvt_temp_public_id",
+  "type": "TAG",
+  "version": 1,
+  "brand": {
+    "displayName": "",
+    "id": "brand_dummy"
+  },
+  "containerContexts": [
+    "WEB"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "macrosInSelect": false,
+    "selectItems": [
+      {
+        "displayValue": "Beacon",
+        "value": "beacon"
+      },
+      {
+        "displayValue": "Product Context",
+        "value": "productContext"
+      },
+      {
+        "displayValue": "Event - Add To Cart",
+        "value": "a2c"
+      },
+      {
+        "displayValue": "Event - Order Confirm",
+        "value": "orderConfirm"
+      },
+      {
+        "displayValue": "Event - User Login",
+        "value": "userLogin"
+      }
+    ],
+    "displayName": "Tag Type",
+    "defaultValue": "beacon",
+    "simpleValueType": true,
+    "name": "type",
+    "type": "SELECT"
+  },
+  {
+    "displayName": "Beacon",
+    "name": "Beacon",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "Customer Key can be found under Developer Resources tab in CEC",
+        "displayName": "Customer Key",
+        "simpleValueType": true,
+        "name": "ckey",
+        "type": "TEXT"
+      },
+      {
+        "macrosInSelect": false,
+        "selectItems": [
+          {
+            "displayValue": "Development",
+            "value": "uat"
+          },
+          {
+            "displayValue": "Production",
+            "value": "prod"
+          }
+        ],
+        "displayName": "Environment",
+        "defaultValue": "prod",
+        "simpleValueType": true,
+        "name": "env",
+        "type": "SELECT"
+      }
+    ]
+  },
+  {
+    "displayName": "Product Context",
+    "name": "productContext",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "Comma separated rfkids. Products Context will be applied to all Reflektion widgets in the page, if nothing is specified. Example: rfkid_1,rfkid_2",
+        "displayName": "rfkids",
+        "simpleValueType": true,
+        "name": "rfkids",
+        "type": "TEXT"
+      },
+      {
+        "help": "Create a variable, populate it with list of SKUs in the cart page and use the variable here",
+        "displayName": "SKU List",
+        "simpleValueType": true,
+        "name": "skuList",
+        "type": "TEXT"
+      }
+    ]
+  },
+  {
+    "displayName": "Event - Add To Cart",
+    "name": "a2c",
+    "groupStyle": "ZIPPY_CLOSED",
+    "type": "GROUP",
+    "subParams": [
+      {
+        "help": "pdp OR home OR category OR cart OR qview",
```

Comment:
typo at view

Reply:
it is qview in the API doc (for quick view)